item.rb

```diff
@@ -0,0 +1,18 @@
+class Item
+
+  attr_reader :name, :price, :weight
+  attr_writer :name, :price, :weight
+  
+  def initialize (name, price, weight)
```

Comment:
```suggestion
  def initialize(name, price, weight)
```
если пишем аргументы в скобках то между названием методом и списком аргументов пробела быть не должно